api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters/solutions-output/most-different.service.ts

```diff
@@ -92,4 +94,39 @@ export class MostDifferentService {
     cluster: SolutionPerClusterGroup<ResultRow | undefined>,
   ): cluster is SolutionPerClusterGroup =>
     cluster.every((element) => isDefined(element));
+
+  #getJaccardBinaryDistance = (setA: number[], setB: number[]): number => {
+    if (setA.length !== setB.length) throw new Error('Need to be equal length');
```

Comment:
in theory we should never get empty sets, but just in case, should we throw an error in this case too? at a quick glance (I may be wrong) such situation may lead to a division by zero on the return statement (which could be ok too - either way is a situation that shouldn't be allowed)

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/629/files#diff-6d118bda04a900bbf9d5a32482a01bbe56e1972c45b6cfd4fb1fc8c21158ce76R26-R42

also there are some (possibly) related things to verify.